image.go

```diff
@@ -0,0 +1,9 @@
+package spotify
+
+// Image represents the ImageObject struct in the API
```

Comment:
Stay consistent with your other comments

```suggestion
// Image represents the ImageObject struct in the Spotify API
```